gigascience/gigadb-website
# Admin styles - Admin Dataset Create Update Page

# Pull request for issue: #1465 

This is a pull request for the following functionalities:

* Admin Dataset create (http://gigadb.gigasciencejournal.com:9170/adminDataset/create) and update (http://gigadb.gigasciencejournal.com:9170/adminDataset/update/id/8/) pages updated to more modern look

Before

![image](https://github.com/gigascience/gigadb-website/assets/143437854/84a75847-8b18-4a72-b104-5078f8ea69bd)


After

![image](https://github.com/gigascience/gigadb-website/assets/143437854/3ea8a3e6-8d6d-44d5-adbb-28ddc2c4b127)


Alternative without wells

![image](https://github.com/gigascience/gigadb-website/assets/143437854/c40d7cd4-c202-4aa1-9c8f-6585dceca4d4)


## How to test?

* Navigate to each of the above pages, confirm that the page uses modern layout and look, interact with the form and confirm the behavior is as expected
* Optionally provide feedback on preferred layout

## How have functionalities been implemented?

* Replaced old layout by modern layout
* Reworked dataset form styles
* Cherry-picked reusable code from other feature branches (i.e jQuery polyfill, title breadcrumb)

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
